src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
```

Comment:
Again..I did not touch it so my merge was bad or Intellij done something :) Anyways...good eye!